src/redist/targets/GenerateLayout.targets

```diff
@@ -66,11 +67,6 @@
       <CoreSetupDownloadDirectory>$(IntermediateDirectory)/coreSetupDownload/$(MicrosoftNETCoreAppPackageVersion)</CoreSetupDownloadDirectory>
       <CombinedSharedHostAndFrameworkArchive>$(CoreSetupDownloadDirectory)/combinedSharedHostAndFrameworkArchive$(ArchiveExtension)</CombinedSharedHostAndFrameworkArchive>
     </PropertyGroup>
-
-    <PropertyGroup Condition="'$(HostOSName)' == 'win'">
-      <AlternateAppHostRid Condition="'$(Architecture)' == 'x86'">win-x64</AlternateAppHostRid>
-      <AlternateAppHostRid Condition="'$(Architecture)' == 'x64'">win-x86</AlternateAppHostRid>
-    </PropertyGroup>
```

Comment:
Hmm, we are taking advantage of having both apphosts in zip for consumption in VS build. I was hoping this would still be possible here. But I realize now that this would probably mean 4 installers for windows instead of 2. 

To avoid breaking anytrhing, can we keep both of the packages in x86 and x64 zips, but only the corresponding bitness in installers?

Reply:
Done.